### PR TITLE
Release 10/11/23

### PR DESCRIPTION
**Pull request by sycombs**
The following packages will be released:

  - terra-application-layout@5.75.0
  - terra-application-links@6.71.0
  - terra-application-navigation@1.73.0
  - terra-collapsible-menu-view@6.81.0
  - terra-data-grid@0.8.0
  - terra-date-input@1.46.0
  - terra-date-picker@4.92.0
  - terra-date-time-picker@4.98.0
  - terra-dialog-modal@3.94.0
  - terra-form-validation@1.89.0
  - @cerner/terra-framework-docs@1.40.0
  - terra-menu@6.79.0
  - terra-navigation-prompt@1.77.0
  - terra-navigation-side-menu@2.48.0
  - terra-notification-dialog@4.35.0
  - @cerner/terra-pills@1.14.0
  - terra-popup@6.72.0
  - terra-slide-panel-manager@5.81.0
  - terra-tabs@7.10.0
  - terra-time-input@4.58.0

### CODE REVIEW

File: packages/terra-menu/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-menu",
   "main": "lib/Menu.js",
-  "version": "6.78.0",
+  "version": "6.79.0",
```
Comment (adoroshk) [inline packages/terra-menu/package.json:4]: Correct me if I am wrong, but I had an impression that fixes are patches?

File: packages/terra-popup/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-popup",
   "main": "lib/Popup.js",
-  "version": "6.71.0",
+  "version": "6.72.0",
```
Comment (adoroshk) [inline packages/terra-popup/package.json:4]: The question same as above - if fixes are patches. Please disregard if they are supposed to be minors

File: packages/terra-menu/package.json (hunk 1 above)
Comment (sycombs) [reply packages/terra-menu/package.json:4]: Yes both of these you pointed out should be patch versions, good catch!

Comment (sycombs) [reply packages/terra-menu/package.json]: [59e4ab1](https://github.com/cerner/terra-framework/pull/1832/commits/59e4ab1088afe400e2915ca480cb7a13ac1a3964)

File: packages/terra-popup/package.json (hunk 2 above)
Comment (sycombs) [reply packages/terra-popup/package.json]: [59e4ab1](https://github.com/cerner/terra-framework/pull/1832/commits/59e4ab1088afe400e2915ca480cb7a13ac1a3964)

**Review by adoroshk — APPROVED** [review]

**Review by sdadn — APPROVED** [review]

**Pull request merged**